### ISSUE TITLE
License Change Consent (from MIT to LGPL-3.0 or later)

### ISSUE THREAD

**Issue by EliteAsian123** [YARC-Official/YARG.Core#115]
### YARG.Core will soon be changing it's license from MIT to LGPL-3.0 (or any version later)

LGPL is a more restrictive license compared to MIT, however it's fully compatible. See details [here](https://www.tldrlegal.com/license/gnu-lesser-general-public-license-v3-lgpl-3).

Before proceeding with this license update, we want to ensure that we have your consent. If you are in agreement with the proposed change, **__please reply to this message with your approval__**. If you have any concerns or objections, we value your feedback and would be happy to discuss further.

Your understanding and cooperation are highly appreciated as we work towards the continued growth of YARG. Thank you for your ongoing support!

### Why?

Some of the reasons are: LGPL requires that any modified version of the software (in this case, YARG.Core), must also have their source code published as LGPL. LGPL also has provisions related to patents, which prevents (to an extent) the use of patents to restrict the use of the software (please read the LGPL license for more details regarding these things if interested), among other things.

### Will this affect any plans with the project, or any libraries?

Nope. LGPL will still allow us to do things such as close-sourcing certain parts of the project (such as anti-cheat), and will still allow use to use libraries with proprietary licenses (such as BASS).

### Who?

95% (which in this case is everybody) of everyone who has made code and art contributions to YARG.Core must give consent. Those people are as follows:
* ✅ @TheNathannator 
* ✅ @RileyTheFox
* ✅ @sonicfind 
* ✅ @raphaelgoulart 
* ✅ @TheFatBastid 
* ✅ @jgoyvaerts 
* ✅ @JayDiddyThaGOAT 

**YARG will also soon have a similar change which will require separate consent.**

Any objections from these people will be worked out before this change is made. Thanks for your contributions and understanding.

### TL;DR

Please give approval below of whether you not you consent of YARG.Core changing licenses from MIT to LGPL-3.0 (or any version later) if you have been mentioned above.

**Comment by raphaelgoulart** [comment]
I consent to YARG.Core changing its license.

**Comment by TheNathannator** [comment]
I fully consent and endorse this change 👍

**Comment by TheFatBastid** [comment]
I consent to this change. Rock on!

**Comment by RileyTheFox** [comment]
I consent to this change.

**Comment by JayDiddyThaGOAT** [comment]
I consent to this change!

**Comment by jgoyvaerts** [comment]
Consent

**Comment by sonicfind** [comment]
Consent
